stdlib/LinearAlgebra/src/matmul.jl

```diff
@@ -581,20 +596,22 @@ function herk_wrapper!(C::Union{StridedMatrix{T}, StridedMatrix{Complex{T}}}, tA
     return gemm_wrapper!(C, tA, tAt, A, A, _add)
 end
 
-function gemm_wrapper(tA::AbstractChar, tB::AbstractChar,
+Base.@constprop :aggressive function gemm_wrapper(tA::AbstractChar, tB::AbstractChar,
```

Comment:
The `constprop` annotation is necessary to get the `wrap` on line 610 to be concretely inferred. The value of `tA` should be propagated to `wrap`.

Reply:
Can you add a comment in the code, so that others will see that the annotation was added deliberately?

Reply:
Done